lib/y2lan_restart_common.pm

```diff
@@ -407,9 +408,16 @@ If network is controlled by Networkmanager, don't change any network settings.
 =cut
 
 sub open_yast2_lan {
-    my $is_nm = !script_run('systemctl is-active NetworkManager');    # Revert boolean because of bash vs perl's return code.
-
-    $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'lan');
+    my %options = @_;
+    my $is_nm
+      = !script_run('systemctl is-active NetworkManager');    # Revert boolean because of bash vs perl's return code.
+    my $y2_opts
+      = ($options{ui} =~ /ncurses/)
+      ? "--" . $options{ui}
+      : "";
+    $y2_opts = "--ncurses" if is_pvm_hmc();
```

Comment:
same to the above, maybe it is better without "()"

Reply:
see above feedback.